demos/python_demos/deep_speech_demo/speech_features.py

```diff
@@ -0,0 +1,199 @@
+##################################################
+#
+# John Feng, 2019/09/06 
+#
+##################################################
```

Comment:
Please remove these banners and add the standard OMZ copyright banners instead. (You already have the correct banner in the main script.)